frontend/simple/views/Invite.vue

```diff
@@ -128,6 +228,13 @@ export default {
       } else {
         this.self = false
       }
+      // Check if this user has been added already
+      if (this.contract.members.find(member => member === this.searchUser) ||
+        this.contract.invitees.find(invitee => invitee === this.searchUser) ||
+        this.candidateMembers.find(username => username === this.searchUser)) {
+        this.error = true
```

Comment:
`this.candidateMembers` and `this.error` look like they're part of the same grouping / logical construction because of the indentation. If you can put the closing parenthesis and the brace on their own line it will make the separation clear, i.e.:

```js
      if (this.contract.members.find(member => member === this.searchUser) ||
        this.contract.invitees.find(invitee => invitee === this.searchUser) ||
        this.candidateMembers.find(username => username === this.searchUser)
      ) {
        this.error = true
        return
      }
```